easybuild/easyconfigs/o/OpenMMTools/OpenMMTools-0.23.1-foss-2023a.eb

```diff
@@ -0,0 +1,78 @@
+easyblock = 'PythonBundle'
+
+name = 'OpenMMTools'
+version = '0.23.1'
+
+homepage = 'https://github.com/choderalab/openmmtools'
+description = """A batteries-included toolkit for the GPU-accelerated OpenMM molecular simulation engine.
+openmmtools is a Python library layer that sits on top of OpenMM to provide access to a variety of useful tools 
+for building full-featured molecular simulation packages.
+"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'usempi': True}
+
+builddependencies = [
+    ('hatchling', '1.18.0'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('OpenMM', '8.0.0'),
+    ('netcdf4-python', '1.6.4'),
+    ('MDTraj', '1.9.9'),
+    ('PyYAML', '6.0'),
+    ('numba', '0.58.1'),
+]
+
+use_pip = True
+exts_list = [
+    ('ipython_genutils', '0.2.0', {
```

Comment:
`ipython_genutils`, `jupyter_core` and `traitlets` can be replaced by a dependency on `jupyter-server`